Source/Scene/Points3DTileContent.js

```diff
@@ -148,19 +155,42 @@ define([
         }
     });
 
+    function createFeatures(content) {
```

Comment:
Yes it's implemented in my latest changes. I'll make sure to include this in the spec and also add a comment in the code.